ddsrecorder/src/cpp/writer/implementations/recorder/TypeObjectWriter.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+    // Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
Remove space. 

I didn't check the copyright year in the first review. Should we change it in all files? If that's the case, I'd to it in another PR as that commit will modify every file

Reply:
Fair enough 👍🏼